ui/components/app/snaps/snap-authorship-header/snap-authorship-header.js

```diff
@@ -33,12 +32,10 @@ const SnapAuthorshipHeader = ({
   // like it is done with snap install
   const packageName = snapId && stripSnapPrefix(snapId);
 
-  const subjectMetadata = useSelector((state) =>
-    getTargetSubjectMetadata(state, snapId),
+  const { name: snapName } = useSelector((state) =>
+    getSnapMetadata(state, snapId),
   );
 
```

Comment:
Can this be uniform ? Sometimes you renamed it to `snapName` and sometimes `friendlyName`

Reply:
Yup, will make it consistent with `snapName`